packages/database/src/migrations/20201009061046-UpdateLaosEocEntities-modifies-data.js

```diff
@@ -0,0 +1,257 @@
+'use strict';
+
+import { arrayToDbString, generateId, insertObject } from '../utilities';
+import { insertEntity } from '../utilities/migration';
+import { EntityModel } from '../modelClasses';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+/*
+ * Each province has a capital and each capital has a point center lat/lon.
+ * See: https://www.wikiwand.com/en/Administrative_divisions_of_Laos
+ *
+ * Each province already has bounds set in the database.
+ */
+const PROVINCES = {
+  Attapeu: {
+    capital: {
+      name: 'Xaisettha',
+      lat: 14.82,
+      lon: 106.820556,
+    },
+  },
+  Bokeo: {
+    capital: {
+      name: 'Houayxay',
+      lat: 20.263056,
+      lon: 100.433611,
+    },
+  },
+  Bolikhamsai: {
+    capital: {
+      name: 'Paksan',
+      lat: 18.396389,
+      lon: 103.655833,
+    },
+  },
+  Champasak: {
+    capital: {
+      name: 'Pakse',
+      lat: 15.116667,
+      lon: 105.783333,
+    },
+  },
+  Houaphanh: {
+    capital: {
+      name: 'Sam Neua',
+      lat: 20.415,
+      lon: 104.048,
+    },
+  },
+  Khammouane: {
+    capital: {
+      name: 'Thakhek',
+      lat: 17.4,
+      lon: 104.8,
+    },
+  },
+  'Luang Namtha': {
+    capital: {
+      name: 'Namtha',
+      lat: 20.95,
+      lon: 101.4,
+    },
+  },
+  'Luang Prabang': {
+    capital: {
+      name: 'Luang Prabang',
+      lat: 19.883333,
+      lon: 102.133333,
+    },
+  },
+  Oudomxay: {
+    capital: {
+      name: 'Xai',
+      lat: 20.691389,
+      lon: 101.986111,
+    },
+  },
+  Phongsaly: {
+    capital: {
+      name: 'Phongsali',
+      lat: 21.683333,
+      lon: 102.1,
+    },
+  },
+  Xainyabuli: {
+    capital: {
+      name: 'Xainyabuli',
+      lat: 19.416667,
+      lon: 103.166667,
+    },
+  },
+  Salavan: {
+    capital: {
+      name: 'Salavan',
+      lat: 15.716667,
+      lon: 106.416667,
+    },
+  },
+  Savannakhet: {
+    capital: {
+      name: 'Kaysone Phomvihane',
+      lat: 16.55,
+      lon: 104.75,
+    },
+  },
+  Xekong: {
+    capital: {
+      name: 'Lamarn',
+      lat: 15.344444,
+      lon: 106.7175,
+    },
+  },
+  'Vientiane Capital': {
+    capital: {
+      name: 'Sisattanak',
+      lat: 17.93219235318203,
+      lon: 102.63719558715822,
+    },
+  },
+  'Vientiane Province': {
+    capital: {
+      name: 'Phônhông',
+      lat: 18.5,
+      lon: 102.416667,
+    },
+  },
+  Xiangkhouang: {
+    capital: {
+      name: 'Pek',
+      lat: 19.416667,
+      lon: 103.166667,
+    },
+  },
+  Xaisomboun: {
+    capital: {
+      name: 'Anouvong',
+      lat: 18.9057,
+      lon: 103.092,
+    },
+  },
+};
+
+// Pre-calculate entity codes (needed for both up() and down()), and id
```

Comment:
Perhaps this could go in `up`? Since you are deleting by regex you don't need it in `down` and personally I wouldn't expect any code to be run at top level when reading a migration. Definitely just an opinion though.